chat/fe/src/views/RoomView.ts

```diff
@@ -360,25 +361,61 @@ export function RoomView(props: RoomViewProps) {
     });
 
     async function init() {
+      const pageSize = 50;
+
       const messagesList = await getRoomMessages(props.roomId);
```

Comment:
The biggest aspect of pagination is ensuring the API only returns the number of messages we are rendering.  In this case, it's fetching all of the messages and then slicing off 50 messages from there.  Instead we want to fetch only 50 messages and only on clicking the load more should it return the next 50 messages.